### ISSUE TITLE
check_ping runs 30 times when host is down

### ISSUE THREAD

**Issue by rcknag** [nagios-plugins/nagios-plugins#139]
When a host is down, check_ping will execute 30 times disregarding what value is set by -p. When the host is up, it is fine.

See this thread for a full tcpdump -
https://support.nagios.com/forum/viewtopic.php?f=7&t=36682&p=172260#p172340

**Comment by jfrickson** [comment]
Fixed in branch 'maint' via commit https://github.com/nagios-plugins/nagios-plugins/commit/b7ee01a257d0bee30b3ef4d579ad56533f389e70

The `ping` command has both a `-w` and `-W` option. Lower-case `w` (which was being used) effectively ignores the `-c` parameter. So it has been changed to an upper-case `W` which will stop sending pings and just wait for the timeout.

**Comment by bardahlm** [comment]
This change broke my checks. All checks on hosts that are down now report 
`CRITICAL - Plugin timed out`
instead of 
`PING CRITICAL - Packet loss = 100%|rta=500.000000ms;300.000000;500.000000;0.000000 pl=100%;20;25;0`

If I set -W 15, ping still takes 24 seconds to terminate:

> $ time /usr/bin/ping -n -U -W 15 -c 10 10.10.10.10
> PING 10.10.10.10 (10.10.10.10) 56(84) bytes of data.
> 
> --- 10.10.10.10 ping statistics ---
> 10 packets transmitted, 0 received, 100% packet loss, time 9001ms
> 
> 
> real    0m24.004s
> user    0m0.002s
> sys     0m0.001s

If I use -w 15, it terminates in just over 15 seconds:

> $ time /usr/bin/ping -n -U -w 15 -c 10 10.10.10.10
> PING 10.10.10.10 (10.10.10.10) 56(84) bytes of data.
> 
> --- 10.10.10.10 ping statistics ---
> 15 packets transmitted, 0 received, 100% packet loss, time 14001ms
> 
> 
> real    0m15.002s
> user    0m0.001s
> sys     0m0.003s
>
